src/modules/openrccfg/module.desc

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Most modules have some documentation explaining what the module does, above this --- comment line.